release_notes/371-release-notes.adoc

```diff
@@ -0,0 +1,225 @@
+| DEP
+| DEP
+| REM
+
+|`/v1/nodes` and `/v1/images` API endpoint response: `firstNodeOccurrence` response body parameter
+|DEP
+|DEP
+|REM
+
+
+| Permissions: `AuthPlugin`, `AuthProvider`, `Group`, `Licenses`, `Role`, `User`, `Indicator`, `NetworkBaseline`, `ProcessWhitelist`, `Risk`, `APIToken`, `BackupPlugins`, `ImageIntegration`, `Notifier`, `SignatureIntegration`, `ImageComponent`
+| GA
+| GA
+| DEP
+
+| Retrieving groups by property
+| GA
+| GA
+| DEP
+
+|====
+
+[id="deprecated-features-371"]
+=== Deprecated features
+
+* Permissions for permission sets will be grouped for simplification. The following list describes the new permissions and indicates the deprecated permissions that will be removed in a future release:
+** New permission Access will replace the permissions `AuthPlugin`, `AuthProvider`, `Group`, `Licenses`, `Role`, and `User`.
+** New permission `DeploymentExtension` will replace the permissions `Indicator`, `NetworkBaseline`, `ProcessWhitelist`, and `Risk`.
+** New permission Integration will deprecate the permissions `APIToken`, `BackupPlugins`, `ImageIntegration`, `Notifier`, and `SignatureIntegration`.
+** New permission Image will replace the permission `ImageComponent`.
+* Previously, groups were retrieved by the field props (`props.authProviderId`, `props.key`, `props.value`). This field will be replaced by the new `props.id` field. Use the props.id field to retrieve groups in the Application Programming Interface (API). Note the following:
+** Retrieval via the `props` fields will be removed in version {product-title-short} version 3.73.
+** Until removal, retrieval via the `props` field will work as long as the result is unambiguous (no more than one group is found with the  props field).
+
+
+[id="removed-features-371"]
+=== Removed features
+
+* *Anchore*, *Tenable*, and *Docker Trusted Registry* integrations: The {product-title-short} scanner supersedes these integrations.
+* *External authorization plug-in for scoped access control*: Use the existing in-product scoped access control.
+* *FROM option in the Disallowed Dockerfile line policy field*: Any policies containing the Disallowed Dockerfile line policy field with the `FROM` option must be updated to remove those policy sections.
+* PodSecurityPolicies (PSPs) can be disabled when generating deployment bundles and when configuring the Helm charts. The Helm charts also support auto-sensing availability of the PodSecurityPolicies API. You must disable PSPs when deploying to Kubernetes version 1.25 or later.
+
+
+[id="bug-fixes-371"]
+== Bug fixes
+
+* Updated the node affinity setting in Helm templates to discourage use of {product-title-short} pods such as Central, Sensor, Scanner, and Scanner-db being scheduled on control plane nodes. (*ROX-11533*)
+* Because the `FROM` option in the Disallowed Dockerfile line policy field has been deprecated, removed the option to create or edit policies that include this field. (*ROX-10925*)
```

Comment:
Suggestion: 

* Because the `FROM` option in the Disallowed Dockerfile line policy field has been deprecated, this update removed the option to create or edit policies that include this field. (*ROX-10925*)